pkg/events/recorder/recorder.go

```diff
@@ -67,7 +68,12 @@ func (sr *SimpleRecorder) makeEvent(ref *v1.ObjectReference, labels map[string]s
 	t := metav1.NewTime(now)
 	namespace := ref.Namespace
 	if namespace == "" {
-		namespace = metav1.NamespaceDefault
```

Comment:
Do we really need to have this check on every `makeEvent`? Can we init global var with namespace on startup and use it?